client/components/thank-you/index.tsx

```diff
@@ -8,11 +8,13 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import VerticalNav from 'calypso/components/vertical-nav';
-import VerticalNavItem from 'calypso/components/vertical-nav/item';
+import Gridicon from 'calypso/components/gridicon';
 import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
-import { ThankYouNextStepProps, ThankYouProps, ThankYouSectionProps } from './types';
-
+import type {
+	ThankYouNextStepProps,
+	ThankYouProps,
+	ThankYouSectionProps,
+} from 'calypso/components/thank-you/types';
 /**
```

Comment:
Nit: I think this will be easier to understand/read with a trailing newline.
```suggestion
} from 'calypso/components/thank-you/types';

/**
```

Reply:
I did that because I had a lint error in the PHP storm, after removing the empty space it disappeared, but now it is not appearing again 😕 . Removing it anyway 🙂